app/assets/javascripts/shared/detect-scrollbar-width.js

```diff
@@ -0,0 +1,20 @@
+setTimeout(function () {
+    var scrollDiv = document.createElement("div");
+    scrollDiv.className = "scrollbar-measure";
+    document.body.appendChild(scrollDiv);
+    var scrollbarWidth = scrollDiv.offsetWidth - scrollDiv.clientWidth;
+    $('body').append(
+        `<style>
```

Comment:
Hm, according to the HTML standard, the `style` tag must be in the header, not the body. https://developer.mozilla.org/en-US/docs/Web/HTML/Element/style

Maybe it would be better to find the affected elements and apply the style there?

The naming of the file is misleading. It suggests that the width is only detected but the file also changes the layout.

Reply:
Yes, you're right, style should be in the header, let me check if it doesn't affect the workaround (maybe the page must be fully loaded, so i attached it at the end and putted the timeout). 

Will change the name too, you're right, will do the changes as soon as possible, thanks.